### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by viktoriia-kosenko**

### CODE REVIEW

File: client/src/components/Login.js [hunk 1]
```diff
@@ -21,8 +21,15 @@ export class Login extends Component {
   handleSubmit = event => {
     event.preventDefault();
     const { email, password } = this.state;
+
     getToken(email, password)
-      .then(res => sessionStorage.setItem("token", res.token))
+      .then(res => {
+        console.log(res);
```
Comment (llh1) [inline client/src/components/Login.js]: console.log should always be removed, it should only be used for debugging locally only.

File: package.json [hunk 2]
```diff
@@ -14,5 +14,8 @@
     "server/node_modules"
   ],
   "author": "",
-  "license": "ISC"
+  "license": "ISC",
+  "dependencies": {
+    "moment": "^2.24.0"
```
Comment (llh1) [inline package.json]: moment should not be added in the dependencies of the main package.json. Are you using it in the client or in the server? You need to put this dependency in the right package.json (in client or server) and remove it from here.

File: client/src/index.js [hunk 3]
```diff
@@ -86,6 +92,8 @@ const Routes = () => {
           {isLoggedIn() ? (
             <Route path="/add-document/" component={AddDocumentForm} />
           ) : null}
+
+          <Route path="/dashboard" component={Dashboard} />
```
Comment (llh1) [inline client/src/index.js]: The dashboard route should only be active if the user is logged in.

File: client/src/index.js [hunk 4]
```diff
@@ -41,23 +42,28 @@ const Routes = () => {
...
-              Become Interpreter
-            </Menu.Item>
-          </Link>
+          {isLoggedIn() ? null : (
+            <Link className="nav-link" to="/sign-up-interpreter">
+              <Menu.Item name="editorials" active={true}>
+                Become Interpreter
+              </Menu.Item>
+            </Link>
+          )}
```
Comment (llh1) [inline client/src/index.js]: Can you group all these condition into one? It would make the code more maintainable and easier to read :)

**Review by llh1 — CHANGES_REQUESTED** [review]
Great job @kosenchiha 👍  I've added few comments for minor things to fix.

File: client/src/components/Login.js (hunk 1 above)
Comment (viktoriia-kosenko) [reply client/src/components/Login.js]: Removed

File: package.json (hunk 2 above)
Comment (viktoriia-kosenko) [reply package.json]: Added moment to the client side, removed from main

File: client/src/index.js (hunk 3 above)
Comment (viktoriia-kosenko) [reply client/src/index.js]: Changed

**Review by llh1 — APPROVED** [review]

**Pull request merged**